src/Pages/ViewList.js

```diff
@@ -1,23 +1,66 @@
-import React from 'react';
+// import { doc } from 'prettier';
+import React, { useState } from 'react';
 import { useCollection } from 'react-firebase-hooks/firestore';
 import { db } from '../lib/firebase';
 
 const ViewList = ({ token }) => {
   // if (token === null) token = 'default';
 
+  //{someArray.map((val, i) => <input onChange={e => handleInput(e, i)}></input>)}
+  //create  <input> elements and specify their name attributes
+
   const [list, loading, error] = useCollection(db.collection(token));
 
+  //const [filter, setFilter] =useState;
+  const [inputValue, setInputValue] = useState('');
+
+  const handleChange = (e) => {
+    e.preventDefault();
+    setInputValue(e.target.value);
+  };
+
   return (
     <div>
       <h2>Grocery List</h2>
+      <div>
+        <form action="#">
+          <label htmlFor="filter-list" style={{ display: 'hidden' }}>
```

Comment:
Are we hiding this label?

Reply:
Evidently not lol--removing the style :)